src/test/config/bitbucket/values-EKS.yaml

```diff
@@ -7,6 +7,16 @@ ingress:
   create: true
   host: ${helm.release.prefix}-bitbucket.${eks.ingress.domain}
 
+bitbucket:
+  sshService:
+    enabled: true
+    annotations:
+      external-dns.alpha.kubernetes.io/hostname: ${helm.release.prefix}-bitbucket-ssh.${eks.ingress.domain}
+      service.beta.kubernetes.io/aws-load-balancer-internal: 'true'
+  additionalEnvironmentVariables:
+    - name: PLUGIN_SSH_BASEURL
```

Comment:
I believe this is the [system property](https://confluence.atlassian.com/bitbucketserver/configuration-properties-776640155.html#Configurationproperties-plugin.ssh.baseurl) supported by BB. This something customers need to configure as well in case they are using separate domain for ssh (and they don't want to configure it manually). Is this something we will have in the documentation?

Reply:
This is only necessary if you need automatic setup. You can always set this manually in the UI. I think we have a documentation for `bitbucket.properties` somewhere, where this property is documented.

Here it is: https://confluence.atlassian.com/bitbucketserver/configuration-properties-776640155.html#Configurationproperties-plugin.ssh.baseurl

Reply:
I'll make a note of it in the docs page for this service.